cime_config/machines/config_workflow.xml

```diff
@@ -53,11 +53,12 @@
       <runtime_parameters>
         <walltime>0:30:00</walltime>
       </runtime_parameters>
-    </job>
+    </job-->
     <job name="case.st_archive">
       <template>template.st_archive</template>
       <!-- If DOUT_S is true and case.run (or case.test) exits successfully then run st_archive-->
-      <dependency>(case.run and case.post_run_io) or case.test</dependency>
+      <!--dependency>(case.run and case.post_run_io) or case.test</dependency-->
+      <dependency>case.run or case.test</dependency>
```

Comment:
A CIME config change
This is a SCREAM repo only change for now. This change disables the ADIOS to NetCDF conversion job from running after the E3SM job completes.

We will need to add a way to disable it via XML changes in E3SM.